app/(with-main-header)/albatalk/[talkId]/_components/CommentList.tsx

```diff
@@ -104,7 +105,7 @@ const CommentList = ({
                   </div>
                   <div className="text-gray-300">|</div>
                   <div className="max-w-40 text-gray-500 text-xs md:text-md lg:text-lg font-regular">
-                    {format(new Date(comment.createdAt), 'yyyy.MM.dd')}
+                    {formatDate(new Date(comment.createdAt))}
```

Comment:
오 채택됐다 ㅋㅋㅋ
이거 new Date로 안넘겨도 돼요! 그냥 string 이러케만 넣어도 동작합니다!
```suggestion
{formatDate(comment.createdAt)}
```

Reply:
앜 그렇군요!!! 다음 pr때 반영해보겟씀다